class-work/src/task/navigation/LoginPage.java

```diff
@@ -0,0 +1,31 @@
+package task.navigation;
+
+import java.util.Scanner;
+
+public class LoginPage {
+
+    Scanner scanner = new Scanner(System.in);
+
+    private final String USER_NAME = "Oleksiy";
+    private final String USER_PASSWORD = "123456";
+
+
+    public void setName() {
```

Comment:
Login logic can be bypassed with wrong credentials. all the other methods are invoked even if credentials are wrong, pls think on logic reimplementation :)